src/e2e/pages/landingPage.js

```diff
@@ -12,6 +12,17 @@ export class LandingPage {
     this.howItWorksSec = page.locator('.how-it-works')
     this.questionsAboutSec = page.locator('.top-questions-about-monitor')
     this.seeIfDataBreachSec = page.locator('.see-if-data-breach')
+    this.MozillaFooterLogoImage = page.getByAltText("Mozilla")
+    this.MozillaFooterLogoLink = page.locator('footer a').first()
```

Comment:
This seems to match any link in the footer, not necessarily that Mozilla logo. I think you should be able to use the alt text as the name here as well, assuming that `name` is the accessible name/

Reply:
To answer your concern, Maintenance and Reliability were all considered before these scripts are created. as part of an upcoming ticket, will be add labels to these tests as we'll need to run only the core flows during PRs however, we'll need to automate the automate-ables from the QA regression